src/app/components/pool-size-chart/pool-size-chart.component.html

```diff
@@ -1 +1,5 @@
 <div #chart id="stats"></div>
+<div class="pool-size-chart-controls">
+    <button #linearScaleButton disabled="true" (click)="setLogScale(false)">Linear</button>
+    <button #logScaleButton (click)="setLogScale(true)">Logarithmic</button>
```

Comment:
Would be cool to use our style of buttons. Try `button button-blue` classes. Perhaps you could also introduce `button-small` or `button-gray`.